CMakeLists.txt

```diff
@@ -706,39 +788,49 @@ string(REGEX REPLACE "([^;]+)" "find_dependency(\\1)" _find_dependency_calls "${
 string(REPLACE ";" "\n" _find_dependency_calls "${_find_dependency_calls}")
 
 write_basic_package_version_file(
-	"${CMAKE_CURRENT_BINARY_DIR}/libtorrent-rasterbar/libtorrent-rasterbarConfigVersion.cmake"
+	"${CMAKE_CURRENT_BINARY_DIR}/libtorrent-rasterbar/libtorrent-rasterbar-config-version.cmake"
 	VERSION ${libtorrent_VERSION}
-	COMPATIBILITY ExactVersion
+	COMPATIBILITY SameMajorVersion
```

Comment:
There should be `SameMinorVersion` here, AFAIK, but that option is available in cmake >= 3.11 only.